docs/src/site/twiki/EntitySLAMonitoring.twiki

```diff
@@ -0,0 +1,25 @@
+---++Falcon Entity SLA Monitoring
+
+Entity SLA monitoring allows you to monitor the entity(process and feed).It keeps track of the instances of the entity that are running and stores them in the db.
```

Comment:
From @sepreece's comment, the spacing before '(' and after '.' is still not addressed.